### PR TITLE
Complete nav bar links. Add missing management page.

### PR DESCRIPTION
**Pull request by marcorosas1991**

### CODE REVIEW

File: src/sidenav/SideNav.js [hunk 1]
```diff
@@ -30,9 +30,10 @@ class SideNav extends React.Component {
         <Paper className="side-nav-paper">
           <MenuList className="side-nav-list">
             {this.renderMenuItem("Translations", ROUTES.TRANSLATIONS)}
-            {this.renderMenuItem("Settings", ROUTES.SETTINGS)}
+            {this.renderMenuItem("User contributions", ROUTES.CONTRIBUTIONS)}
```
Comment (parikhshiv) [inline src/sidenav/SideNav.js]: "User Contributions" (second word capitalized, to match other nav cases)

File: src/App.js [hunk 2]
```diff
@@ -15,6 +15,7 @@ const ROUTES = {
   SETTINGS: '/settings',
```
Comment (parikhshiv) [inline src/App.js]: seems like you want to remove settings here?

Comment (marcorosas1991) [reply src/App.js]: yes

File: src/sidenav/SideNav.js (hunk 1 above)
Comment (marcorosas1991) [reply src/sidenav/SideNav.js]: Done

**Pull request merged**